### DISCUSSION TITLE
K1 Max camera reload macro fails

### DISCUSSION THREAD

**Discussion by katanabladesman** [Guilouz/Creality-Helper-Script-Wiki#601] [Issues]
I have the usb camera option installed using the script. Sometimes the cameras (internal and usb) fail and only come back up after reboot. I have tried the "reload camera" macro and it does not work. Any tips or ideas?

// Running Command {Reload_Camera}...:
// Info: Reloading camera config...
// start-stop-daemon: warning: killing process 10117: No such process
// start-stop-daemon: warning: killing process 10132: No such process
// sh: V20101: unknown operand
// start-stop-daemon: warning: killing process 10146: No such process
// start-stop-daemon: warning: killing process 10153: No such process
// sh: 230822: unknown operand
// Command {Reload_Camera} finished

**Comment by Supabad** [comment]
does it come back after doing /etc/init.d/S50usb_camera restart in ssh of your printer (putty or winscp)

**Reply by katanabladesman** [reply]
Honestly don't remember if I tried that. I've switched to the Simple AF klipper mod and the k1 klipper mod. I have been trying to get a tradrack going and don't even have dual cam right now. If I go back to stock I'll try it to be sure
